tools/hsettp/kvs-cn-tree-get.c

```diff
@@ -18,27 +18,26 @@
 
 #include "utils.h"
 
-#define IGNORE_ME "-"
-
 #pragma GCC visibility push(default)
 
 const char *kvs_cn_tree_get_headers[] = {
-    "T", /* If this index moves, change the free func */
+    "T",
```

Comment:
Is the comment no longer relevant?

Reply:
Correct.  The free function no longer keys on the column to decide if it should call free() or not.  Instead, we call a simple function on each candidate to tell us whether the string comes from our table of "constant" commonly used fixed strings, and free() it if not..